src/next/components/Infopic/Infopic.tsx

```diff
@@ -0,0 +1,159 @@
+import { InfopicProps } from "~/common"
+import Button from "../Button"
+
+const TextComponent = ({
+  title,
+  description,
+  buttonLabel,
+  buttonUrl,
+  className,
+}: Omit<InfopicProps, "sectionIndex" | "imageSrc" | "imageAlt"> & {
+  className?: string
+}) => {
+  return (
+    <div className={`flex flex-col gap-10 ${className ?? ""}`}>
+      <div className="flex flex-col gap-4 sm:gap-6">
+        <h1 className="text-[#333333] text-2xl font-semibold sm:text-4xl">
```

Comment:
why the text color is hardccoded? shouldn't these come in from the brand colors set in tailwind config?

Reply:
[396feb7](https://github.com/isomerpages/isomer-components/pull/17/commits/396feb770d18de876080bc317fc0333d541e966a)